src/docuquery_ai/services/file_service.py

```
@@ -3,7 +3,12 @@
 from typing import Any, Dict, List, Optional
 
-from sqlalchemy.orm import Session
+try:
+    from sqlalchemy.orm import Session
+    SQLALCHEMY_AVAILABLE = True
+except Exception:  # pragma: no cover - optional dependency missing
+    SQLALCHEMY_AVAILABLE = False
+    Session = None  # type: ignore
 
 from docuquery_ai.core.config import settings
 from docuquery_ai.models.db_models import File, User
```

Ruff (0.11.9)

`docuquery_ai.models.db_models.User` imported but unused

Remove unused import: `docuquery_ai.models.db_models.User`

(F401)